code/modules/admin/verbs/spatialagent.dm

```diff
@@ -0,0 +1,48 @@
+/client/verb/spatialagent()
+	set category = "Debug"
+	set name = "Spawn Spatial Agent"
+	set desc = "Spawns a Spatial Agent. Useful for debugging."
+
+	if (!check_rights(R_DEBUG))
+		return
+
+	var/T = get_turf(usr)
+	var/mob/living/carbon/human/sa/sa = new(T)
+	sa.ckey = usr.ckey
```

Comment:
When changing the key like this, you should also change the world view.
`change_view(world.view)`

(taken from:)
```	
if(O.client) 
	O.client.change_view(world.view)```